third_party/2and3/atomicwrites/__init__.pyi

```diff
@@ -5,8 +5,8 @@ def move_atomic(src: AnyStr, dst: AnyStr) -> None: ...
 class AtomicWriter(object):
     def __init__(self, path: Union[Text, bytes], mode: Text = ..., overwrite: bool = ...) -> None: ...
     def open(self) -> ContextManager[IO[Any]]: ...
-    def _open(self, get_fileobject: Callable[..., IO[AnyStr]]) -> ContextManager[IO[AnyStr]]: ...
-    def get_fileobject(self, dir: Union[None, Text, bytes] = ..., **kwargs: Any) -> IO[Any]: ...
+    def _open(self, getFileDescriptorLike: Callable[..., IO[AnyStr]]) -> ContextManager[IO[AnyStr]]: ...
+    def getFileDescriptorLike(self, dir: Union[None, Text, bytes] = ..., **kwargs: Any) -> IO[Any]: ...
```

Comment:
I assume there are replace errors?

Reply:
Yup! Guess I should have ran the tests again